zkevm-circuits/src/evm_circuit/util/math_gadget.rs

```diff
@@ -200,6 +201,268 @@ impl<F: FieldExt, const N: usize> AddWordsGadget<F, N> {
     }
 }
 
+/// Construction of 2 256-bit words mutiplication and result (modulo 2**256),
+/// which is useful for opcode MUL, DIV, SDIV and xxxMOD
+#[derive(Clone, Debug)]
+pub(crate) struct MulWordsGadget<F> {
+    multiplier: util::Word<F>,
```

Comment:
maybe just just `a` and `b` to match the internal variables like `a_digits`, `tmp_a`...